### PR TITLE
Added possibility to create snapshot

### PR DESCRIPTION
**Pull request by yaliza**
only one snapshot you can create.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="by.paranoidandroid.dailyvisualizer">
-
+    <uses-feature android:name="android.hardware.camera"  android:required="true" />
```
Comment (Kalyanova) [inline app/src/main/AndroidManifest.xml]: Why is camera feature required? Maybe it's better to replace it with false, isn't it?
`<uses-feature android:name="android.hardware.camera"  android:required="false" />  `

File: app/src/main/java/by/paranoidandroid/dailyvisualizer/view/Utils/BitmapManager.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
+package by.paranoidandroid.dailyvisualizer.view.Utils;
```
Comment (Kalyanova) [inline app/src/main/java/by/paranoidandroid/dailyvisualizer/view/Utils/BitmapManager.java]: And why has package "Utils" name, not "utils"?)

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (yaliza) [reply app/src/main/AndroidManifest.xml]: I agree with you)

**Review by Kalyanova — APPROVED** [review]
Great work! 👍 
Please, just rename package before commit)

**Review by filak8 — APPROVED** [review]

**Review by ErgerSusha — APPROVED** [review]

**Pull request merged**